docs/topic-guides/system-concepts/scheduling.txt

```diff
@@ -187,8 +187,12 @@ This plugin is also in beta and is not enabled by default. Similar to the cosche
 enabled by setting the ``defaultScheduler`` field to ``preemption``. Autoscaling is not supported
 and Determined can only automatically set labels for GPU experiments.
 
-If using a cluster with tainted nodes or labels, users must specify the tolerations or node
-selectors in the experiment config:
+Determined provides a default priority class, ``determined-medium-priority`` that has a priority of
```

Comment:
Can we keep the discussion of priority separate from the discussion of taints / tolerations? Very distinct topics - easy to miss the other important detail if you know you don't care about one.